sensor/temperature.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "temperature.h"
+
+using namespace std;
+
+vector<int> generateTemperatureValues(int teperatureMaxValue , int teperatureMinValue, int numberOfReadings) {
+  vector<int> temperatureValueList;
+  for (int counter = 0; counter < numberOfReadings; counter++) {
+    int randomValue = rand() % teperatureMaxValue + teperatureMinValue;

```

Comment:
Please correct the variable name(teperatureMaxValue + teperatureMinValue)

Reply:
changes done.